test/test.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-undef */
+import request from 'supertest';
+
+import app from '../app';
+
+describe('GET /party', () => {
+  it('Fetch all political parties records.', (done) => {
+    request(app)
+      .get('/api/v1/party')
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/)
+      .expect(200, done);
```

Comment:
Your test should ascertain that the data returned is equal to data expected for instance `expect('prince', res.body.name)`